### ISSUE TITLE
Unexpected token in Safari 

### ISSUE THREAD

**Issue by jonathantneal** [vanillawc/wc-monaco-editor#1]
Thanks for sharing your work! I’m excited to try it out. It may not be working as expected. I’ll look into why, but here are the details.

- *URL*: https://vanillawc.github.io/wc-monaco-editor/demo/demo.html
- *Browser*: Safari 13.0.2 (15608.2.30.1.1)
- *Error*: `SyntaxError: Unexpected token '='. Expected an opening '(' before a method's parameter list.`

**Comment by jonathantneal** [comment]
Upon further investigation, the issue I am experiencing in Safari is because **instance field declarations** are not yet supported in that browser.

- *Chrome Platform Status*: https://www.chromestatus.com/feature/6001727933251584
- *Safari Status*: https://bugs.webkit.org/show_bug.cgi?id=174212

This can be worked around by putting those declarations in the constructor with context; from:

```js
export class WCMonacoEditor extends HTMLElement {
  __element = null;
  __editor = null;
```

To:

```js
export class WCMonacoEditor extends HTMLElement {
  constructor () {
    super();
    this.__element = null;
    this.__editor = null;
  }
```

And it runs!

**Comment by evanplaice** [comment]
Thanks for taking the time to file this issue. Awesome quality bug report :+1: 

This should be fixed in the 1.0 release. 

I'm going to close this issue for now. If you're still have issues after updating the package, don't hesitate to reopen it.
